web/src/topic_list.ts

```diff
@@ -482,6 +482,7 @@ export function zoom_in(): void {
             // position since we just added some topics to the list which moved user
             // to a different position anyway.
             left_sidebar_scroll_zoomed_in_topic_into_view();
+            add_unresolved_pill_by_default();
```

Comment:
Hm, why do we need to call the function twice? Calling it once on the `setup_topic_search_typeahead` should have been enough?